### PR TITLE
First two pages of New Technologies Conference website

### PR DESCRIPTION
**Pull request by whid-bb**
### First two pages of the new design for NTC:
- Start page
  - animated header section
  - main program points
  - featured speakers created dynamically
  - our partners
- About page
  - animated header with conference description
  - new logo presentation
  - past conferences

Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

Icons used in the project:

- <a href="https://www.flaticon.com/free-icons/rosette" title="rosette icons">Rosette icons created by Good Ware - Flaticon</a>
- <a href="https://www.flaticon.com/free-icons/workshop" title="workshop icons">Workshop icons created by itim2101 - Flaticon</a>
- <div>Icons made by <a href="https://www.flaticon.com/authors/becris" title="Becris">Becris</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
- <div>Icons made by <a href="https://www.flaticon.com/authors/pixelmeetup" title="Pixelmeetup">Pixelmeetup</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
- <div>Icons made by <a href="https://www.flaticon.com/authors/flat-icons" title="Flat Icons">Flat Icons</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
- <div>Icons made by <a href="https://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
- <div>Icons made by <a href="https://www.flaticon.com/authors/uniconlabs" title="Uniconlabs">Uniconlabs</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
- <div>Icons made by <a href="https://www.flaticon.com/authors/ilham-fitrotul-hayat" title="Ilham Fitrotul Hayat">Ilham Fitrotul Hayat</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>

Branch live demo: [demo](https://whid-bb.github.io/capstone-module-one/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,12 +28,6 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
```
Comment (youmari) [inline README.md:69]: - [ ] it would be better if you give credit to the author of the original design

File: index.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                <img src="img/partners/Dell-Logo.png" alt="Dell Logo">
+                <img src="img/partners/Fujitsu-Logo.png" alt="Fujitsu Logo">
+                <img src="img/partners/LG-logo.png" alt="LG Logo">
+            </div>
+        </div>
+    </section>
+    <footer id="page-footer">
+        <div class="content">
+            <div id="footer-content">
+                <h2>New Technologies Conference</h2>
```
Comment (youmari) [inline index.html]: - [ ] It would be better if put the logo here instead of `<h2>` in order to match the design

File: about.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                <div class="lc-img-container">
+                    <div class="lcimg-item">
+                        <img src="images/4e4df3bbabf7db484421.jpg" alt="Image from conference 2020">
+                        <div class="lcm-overlay full-size">
+                            <h4>2020</h4>
+                            <p>NTC in Amsterdam</p>
+                        </div>
+                    </div>
+                    <div class="lcimg-item">
+                        <img src="images/972fc8e9f59af47cf5e0.jpg" alt="Image from conference 2021">
```
Comment (youmari) [inline about.html:131]: - [ ] In the mobile screens the two images of the conferences should be aligned vertically as the mobile design so I encourage you to do that

File: js/speakers.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
+// import Shuntaro from '../img/speakers/Shuntaro-Furukawa.webp';
+// import Sundar from '../img/speakers/Sundar-Pichai.webp';
+// import Susan from '../img/speakers/Susan-Wojcicki.webp';
+// import Tara from '../img/speakers/Tara-Reed.webp';
+// import Tristan from '../img/speakers/Tristan-Harris.webp';
+// import Vanessa from '../img/speakers/Vanessa-Guthrie.webp';
```
Comment (youmari) [inline js/speakers.js]: - [ ] please try to remove the unused commented code to keep your file clean and professional

File: js/gfx-animation.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  return speed !== 0 ? speed : setAttrSpeed();
+};
+
+headGfx.forEach((gfx) => {
+  gfx.setAttribute('data-speed', setAttrSpeed());
+});
+
+header.addEventListener('mousemove', (e) => {
+  headGfx.forEach((gfx) => {
+    // const speed = Math.floor(Math.random() * (-10 - 10) + 10);
```
Comment (youmari) [inline js/gfx-animation.js]: - [ ] please try to remove the unused commented code to keep your file clean and professional

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @whid-bb , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter error
✔️ Great title and short summary
✔️ Nice design personally I really like it 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**